drake_cmake_external/CMakeLists.txt

```diff
@@ -69,7 +69,52 @@ find_package(PythonInterp ${FIND_PYTHON_INTERP_VERSION} MODULE REQUIRED)
 
 include(ExternalProject)
 
+# This shows how to rebuild fmt from source as part of Drake's CMake build.
+# If you'd rather just use your operating system's fmt, then this stanza could
+# be removed (as well as removing -DWITH_USER_FMT:BOOLEAN=ON, below).
+# If you rebuild fmt from source, then you must also rebuild spdlog from source.
+ExternalProject_Add(fmt
+  URL https://github.com/fmtlib/fmt/archive/refs/tags/6.1.2.tar.gz
+  URL_HASH SHA256=1cafc80701b746085dddf41bd9193e6d35089e1c6ec1940e037fcb9c98f62365
+  TLS_VERIFY ON
+  CMAKE_ARGS
+    -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
+    -DCMAKE_C_COMPILER:FILEPATH=${CMAKE_C_COMPILER}
+    -DCMAKE_CXX_COMPILER:FILEPATH=${CMAKE_CXX_COMPILER}
+    -DCMAKE_INSTALL_PREFIX:PATH=${CMAKE_INSTALL_PREFIX}
+    -DCMAKE_PREFIX_PATH:PATH=${CMAKE_PREFIX_PATH}
+    -DCMAKE_VERBOSE_MAKEFILE:BOOL=${CMAKE_VERBOSE_MAKEFILE}
+    -DBUILD_SHARED_LIBS=ON
+    -DCMAKE_CXX_FLAGS:STRING=-fPIC
+    -DFMT_INC_DIR:PATH=include/fmt
+    -DFMT_TEST=OFF
+  BUILD_ALWAYS ON
+  INSTALL_COMMAND :
```

Comment:
This looks suspicious. I am using "installed" (to siblings of my build directory, not anything like `/usr/local`) versions of both fmt and spdlog, as at least one of them was totally unusable from a build tree.

Also, spdlog almost certainly needs `fmt_DIR` specified.